### PR TITLE
Feat/add interface

### PR DESCRIPTION
**Pull request by a-tk**
Adding interface and test method that compares input and btree output, excluding duplicates from the input.

### CODE REVIEW

File: src/test/java/cs321/btree/BTreeTest.java [hunk 1]
```diff
@@ -454,6 +480,62 @@ private boolean validateSearchTree(BTree b) {
...
+     * operations such as splitting children in the tree.
+     *
+     * Also used to validate that all the keys in the BTree
+     * are sorted by using an in order traversal of the tree
+     *
+     * IMPORTANT: handles duplicates in inputKeys by removing them.
+     *
+     * @return true if BTree in order traversal matches provide input
+     */
+    private boolean validateBTreeInserts(BTree b, long[] inputKeys) {
```
Comment (a-tk) [inline src/test/java/cs321/btree/BTreeTest.java:495]: Uglier than intended due to input potentially containing duplicates. Will consider refactoring with more time.
